Source/Cvent.SchemaToPoco.Core/JsonSchemaResolver.cs

```diff
@@ -115,19 +117,26 @@ private JsonSchemaWrapper ResolveSchemaHelper(Uri curr, Uri parent, string data)
             {
                 // Get the full path to the file, and change the reference to match
                 string refName = match.Groups[1].Value;
-                if (!refName.EndsWith(".json"))
-                {
-                    refName += ".json";
+                var currPath = new Uri(refName, UriKind.RelativeOrAbsolute);

```

Comment:
I think this could go below the `ids.Contains` check, unless the idea is to ensure that IDs are valid URIs.